backend/src/schema/resolvers/reports.js

```diff
@@ -5,31 +5,31 @@ export default {
       const { resourceId, reasonCategory, reasonDescription } = params
       const { driver, user } = context
       const session = driver.session()
-      const writeTxResultPromise = session.writeTransaction(async txc => {
-        const reportRelationshipTransactionResponse = await txc.run(
-          `
+      try {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-03T13:11:17Z" title="Tuesday, December 3rd 2019, 2:11:17 pm +01:00">Dec 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -5,31 +5,31 @@ export default {
       const { resourceId, reasonCategory, reasonDescription } = params
       const { driver, user } = context
       const session = driver.session()
-      const writeTxResultPromise = session.writeTransaction(async txc => {
-        const reportRelationshipTransactionResponse = await txc.run(
-          `
+      try {
```


why are you wrapping this in a `try`... `write/read` transaction are `promises`, so they shouldn't need to be wrapped in `try`, oder?